Gems/ROS2/Code/Source/Lidar/ROS2LidarSensorComponent.cpp

```diff
@@ -89,9 +89,20 @@ namespace ROS2
             }
         }
     }
+    AZStd::string GetDeafultLidarSystem(){
+        const auto& lidarSystemList = LidarRegistrarInterface::Get()->GetRegisteredLidarSystems();
+        AZ_Assert(!lidarSystemList.empty(), "Unable to choose a lidarSystem for the lidar sensor.");
```

Comment:
Consider:
"No LIDAR system for the sensor to use" 
Also, assert will be gone in release build so this .front() call will still have a probelm